### ISSUE TITLE
Remove nodes which doesn't have any edges

### ISSUE THREAD

**Issue by sudevschiz** [someshkar/covid19india-network#24]
Removing the singletons will help in focusing only on cases which are part of some cluster.

**Comment by SahajR** [comment]
We can have a filter option for this instead.

**Comment by someshkar** [comment]
Yeah, otherwise the number of patients we show on [the main site](https://www.covid19india.org/) won't match up with the number of nodes we'll be showing on the `P2P` filter, so I think it works for now?
